packages/shared/__tests__/setupTest.js

```diff
@@ -0,0 +1,19 @@
+jest.mock('@booking-services/shared', () => ({
+  __esModule: true,
+  Logger: {
+    info: jest.fn(),
+    error: jest.fn(),
+    warn: jest.fn(),
+    debug: jest.fn(),
+  },
+}));
+
+// TODO add preffixes use to connect to mongoose
+const preffixes = [
+  'messages',
```

Comment:
Here is the part I don't like. 
My function behavior will throw an error when we give a bad prefix (when we don't have that env_var in our file .env.*) so I just add my prefix. The problem is when someone from other team run this test it won't pass. Any suggestions?